README.md

```diff
@@ -17,10 +22,12 @@
 
 ## **Installing** ##
  *install the following dependencies so that the app will work.*
-`npm i node`
-`npm i nodemon`
-`npm i express`
-`npm i txtgen`
+```
+npm i node
+npm i nodemon
+npm i express
+npm i txtgen
```

Comment:
Remove all of that, whenever you install any package, it adds it to the package.json and will be installed automatically for anyone else.
Change it to :

Install all the dependencies:
```bash
npm i
```